wa/workloads/applaunch/__init__.py

```diff
@@ -0,0 +1,132 @@
+#    Copyright 2015 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# pylint: disable=attribute-defined-outside-init
+import os
+
+from wa import Workload, ApkUiautoWorkload, Parameter, ApkWorkload
+from wa import ApkFile, settings
+from wa.framework import pluginloader
+from wa.framework.exception import ConfigError, ResourceError
+
+from devlib.utils.android import ApkInfo
+
+
+class Applaunch(ApkUiautoWorkload):
+
+    name = 'applaunch'
+    description = '''
+    This workload launches and measures the launch time of applications for supporting workloads.
+
+    Currently supported workloads are the ones that implement ``ApplaunchInterface``. For any
+    workload to support this workload, it should implement the ``ApplaunchInterface``.
```

Comment:
This is more of a TO-DO note than a comment (no action required for this pull), but it would be good if we could auto-generate a list of these; maybe with decompiliation/introspection tools in the Android SDK....